src/MinimumNodeService.cpp

```diff
@@ -486,11 +485,4 @@ Processed MinimumNodeService::handleMessage(unsigned int opc, CANFrame *msg)
   }
 }
 
-void MinimumNodeService::setSetupMode()
-{
-  bModeSetup = true;
-  instantMode = MODE_SETUP;
-  timeOutTimer = 0;
-}
-
```

Comment:
Can you bring this back in? 
Seems like you have overwritten an update from me.

Reply:
Sorry about that.  Now back in